app/models/likes.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :updatepostcounter
+
+  private
+
+  def updatepostcounter
+    post.update(likescounter: post.likes.count)
+  end
```

Comment:
- [OPTIONAL] Just like i suggested above, using the `!increment` works better in this scenario. SO I'll suggest you update your method to look something like this. :wave: :waffle: 
```ruby

def update_likes_counter
  post.increment!(:likes_counter)
end
```